cli/azd/pkg/apphost/generate.go

```diff
@@ -43,6 +45,36 @@ func IsAspireDashboardEnabled(alphaFeatureManager *alpha.FeatureManager) bool {
 	return alphaFeatureManager.IsEnabled(aspireDashboardFeature)
 }
 
+type AspireDashboard struct {
+	Link string
+}
+
+func (aspireD *AspireDashboard) ToString(currentIndentation string) string {
+	return fmt.Sprintf("%sAspire Dashboard: %s", currentIndentation, output.WithLinkFormat(aspireD.Link))
+}
+
+func (aspireD *AspireDashboard) MarshalJSON() ([]byte, error) {
+	return json.Marshal(*aspireD)
+}
+
+func AspireDashboardUrl(
+	ctx context.Context,
+	env *environment.Environment,
+	alphaFeatureManager *alpha.FeatureManager) *AspireDashboard {
+	if !IsAspireDashboardEnabled(alphaFeatureManager) {
+		return nil
+	}
+
+	ContainersManagedEnvHost, exists := env.LookupEnv("AZURE_CONTAINER_APPS_ENVIRONMENT_DEFAULT_DOMAIN")
+	if !exists {
+		return nil
+	}
```

Comment:
Right now is just sort of OK because we are using an alpha feature switch. So, having the env var w/o the feature switch won't lead to a false positive. 

I was initially added aspire-full detection (by looking at services and with the importer, but removed that as it was introducing too much complexity here).

In the future, when we remove the alpha feature flag, I am thinking on using cache within the apphost package, so the importer can response faster to isAspire() and then use that later, like in this case

Reply:
Makes sense, thanks for the plan